aws/resource_aws_transfer_server.go

```diff
@@ -150,6 +174,16 @@ func resourceAwsTransferServerCreate(d *schema.ResourceData, meta interface{}) e
 		createOpts.LoggingRole = aws.String(attr.(string))
 	}
 
+	if attr, ok := d.GetOk("certificate"); ok {
+		createOpts.Certificate = aws.String(attr.(string))
+	}
+
+	createOpts.Protocols = []*string{aws.String("SFTP")}
```

Comment:
Although a default protocol is not specified per se, from testing in the AWS Console; the `SFTP` protocol option appears to be the default.

Reply:
Correctly marking the `protocols` attribute as `Optional, Computed` means you don't need to set the default here.